.github/workflows/ci.yml

```diff
@@ -500,6 +507,36 @@ jobs:
           secret_vault_address: ${{ secrets.VAULT_ADDR }}
           secret_vault_roleId: ${{ secrets.VAULT_ROLE_ID }}
 
+  protobuf-checks:
+    name: "Protobuf Backwards Compatibility"
+    timeout-minutes: 10
+    needs: [detect-changes]
+    runs-on: ubuntu-latest
+    if: |
+      needs.detect-changes.outputs.protobuf-changes == 'true' &&
+      contains(fromJSON('["pull_request","push"]'), github.event_name) &&
+      !contains(github.event.pull_request.labels.*.name, 'breaking/protobuf')
```

Comment:
@cmur2 - the assumption here is that this would evaluate to true when the event is not a pull_request, basically. Let me know if you think that's wrong :+1: